src/gridcoin/voting/poll.cpp

```diff
@@ -206,16 +206,83 @@ const Poll::ChoiceList& Poll::Choices() const
     return m_choices;
 }
 
+std::string Poll::PollTypeToString() const
+{
+    return PollTypeToString(m_type.Value());
+}
+
+std::string Poll::PollTypeToString(const PollType& type, const bool& translated)
+{
+    if (translated) {
+        switch(type) {
+        case PollType::UNKNOWN:         return _("Unknown");
+        case PollType::SURVEY:          return _("Survey");
+        case PollType::PROJECT:         return _("Project Listing");
+        case PollType::DEVELOPMENT:     return _("Protocol Development");
+        case PollType::GOVERNANCE:      return _("Governance");
+        case PollType::MARKETING:       return _("Marketing");
+        case PollType::OUTREACH:        return _("Outreach");
+        case PollType::COMMUNITY:       return _("Community");
+        case PollType::OUT_OF_BOUND:    return _("Out of Bound");
```

Comment:
```suggestion
        case PollType::OUT_OF_BOUND:    break;
```
Should not ever be passed to this function in my opinion, and should trigger the assert below.

Reply:
Agreed. I should have changed this from the pre-existing behavior.